src/main/java/space/obminyashka/items_exchange/rest/AdvertisementController.java

```diff
@@ -142,6 +145,21 @@ public AdvertisementModificationDto createAdvertisement(
         return advertisementService.createAdvertisement(dto, owner, compressedImages, scaledTitleImage);
     }
 
+    private void isValidSize(long subcategoryId, String size) {
+        Range<Long> clothingSubcategory = Range.closed(1L, 12L);
+        Range<Long> shoesSubcategory = Range.closed(13L, 17L);
+
+        if (clothingSubcategory.contains(subcategoryId)) { // subcategories ID for Clothing
+            Clothing.fromValue(size);
+        } else if (shoesSubcategory.contains(subcategoryId)) { // subcategories ID for Shoes
```

Comment:
It'd be better to attach both those comments to the ranges initialization above.